src/app/files/browser-files.js

```diff
@@ -49,6 +47,7 @@ function Files (storage) {
         return false
       }
       if (!exists) {
+        this.init()
```

Comment:
I am pretty sure `this.init()` should not be called here (we are in `set()`)

Reply:
yeah, i'll try